_data/matrix/Casa.yaml

```diff
@@ -0,0 +1,36 @@
+---
+name: "Casa"
+category: "Software Wallet (L1)"
+keyfeatures: "High security + UX via multi-vendor multisig"
+homepage: "https://casa.io"
+
+feature:
+  device: "Mobile"
+  os: "Android, iOS"
+  web: "No"
+  hw_interface: "Yes"
+  default_receive: "P2WSH"
+  bech32: "Send Support"
```

Comment:
Something doesn’t add up here. Casa is listed as not having native segwit change, and no receive support for bech32, but their default receive addresses are P2WSH. This is contradictory.

Reply:
Thanks for the catch, I will confirm with the Casa Team.